src/modules/displaytask.js

```diff
@@ -1,16 +1,127 @@
 /* eslint-disable import/prefer-default-export */
-export const displayList = (tasklist) => {
-  for (let i = 0; i < tasklist.length; i += 1) {
-    const element = document.querySelector('.task-lister');
-    element.innerHTML += `
-                <li class="task-item">
-                    <div class="task-input">
-                      <div class="task-div">
-                          <input class="checkbox" type="checkbox"/>
-                          <label class="label">${tasklist[i].description}</label>
-                        </div>
-                        <i class="opt bi bi-three-dots-vertical"></i>
-                    </div>
-                </li> `;
+/* eslint-disable no-restricted-syntax */
+import { toDoListData } from './todo.js';
+import { updateCompletedDisplay } from './updatecomplete.js';
+
+export const displayList = () => {
+  const listItemObject = JSON.parse(window.localStorage.getItem('taskData') || '[]');
```

Comment:
- [ ] I can add tasks and delete but I cannot edit tasks, please add a function that allows users to edit the task. You can use `contenteditable` for that 😉 
```
<div contenteditable="true">
  This text can be edited by the user.
</div>
```

Reply:
Yes. The description of the input can be editable kindly watch the Loom video I have recorded... You can do it clicking the three dots and double click on the description

Reply:
@IamYunusAli , thanks for clarification, because I tried to edit and could not. After double clicking I was able to update and save changes 👍🏻 
OPTIONAL:   it is better to make editing possible on one click I think, to improve UX, or somehow show that users can update the tasks 😃, but this is optional because the functionality works fine